package.json

```diff
@@ -57,6 +57,7 @@
     "@babel/register": "^7.4.0",
     "@canvas-js/core": "0.5.0-alpha4",
     "@istanbuljs/nyc-config-typescript": "^0.1.3",
+    "@openfeature/web-sdk": "^0.4.12",
```

Comment:
do we need this dev dep?

Reply:
It is a dev dependency because it is only required at app build time not runtime (After we have our webpack bundle, we don't need it anymore). By making it a dev dependency, it does not increase our slug size on heroku.